README.md

```diff
@@ -1,4 +1,4 @@
-# शंख Śaṇkha
+# शंख Śaṇkha [![Build Status][travis-badge]][travis][![Dependencies Status][deps-badge]][deps][![Clojars Project][clojars-badge]][clojars][![Clojure version][clojure-v]](project.clj)
 
 *A Divine Shell*
 
```

Comment:
The links below here for `[logo]` and `[logo-large]` need to be moved to the bottom of the file, replacing the logo links that are (incorrect) there.